src/interfaces.ts

```diff
@@ -1,33 +1,11 @@
-import { widgetTypes } from './utils'
-
-export interface WidgetI<T> {
-  title: string
-  widgetType: typeof widgetTypes[number]
-  settings: T
-  minWidth?: number
-}
-export interface MarkdownWidgetI extends WidgetI<{ content: string }> {
-  widgetType: 'markdown'
-}
-
-export interface ReduxMemostateI {
-  type: string
-  value: string
-}
-
-export interface DialogPropsI {
-  open: boolean
-  title: string
-  onClose: (value: boolean) => void
-}
-
-export interface NewMemoI {
+/* eslint-disable @typescript-eslint/interface-name-prefix */
```

Comment:
Can you please remove this comment and rename the interfaces without the I prefix?